interface/modules/custom_modules/oe-module-dorn/public/get_lab_results.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Common\Acl\AclMain;
+    use OpenEMR\Common\Csrf\CsrfUtils;
+    use OpenEMR\Common\Twig\TwigContainer;
+    use OpenEMR\Core\Header;
+    use OpenEMR\Modules\Dorn\ReceiveHl7Results;
+
+if (!empty($_GET)) {
+    if (!CsrfUtils::verifyCsrfToken($_GET["csrf_token_form"])) {
+        CsrfUtils::csrfNotVerified();
+    }
+}
+
+if (!empty($_POST)) {
+    if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
+        CsrfUtils::csrfNotVerified();
+    }
+}
+
+if (!AclMain::aclCheckCore('admin', 'users')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Edit/Add Procedure Provider")]);
+    exit;
+}
```

Comment:
This ACL doesn't seem right to me.
It is used in all place where checked. Does this allow physicians?
Someone who knows more than me may want to comment for correct ACL for lab orders and results.

Reply:
Agreed. Is the pageTitle correct in above line?
What does this script do (view labs etc?). Once know that, would then know what acl check to do (can also see what is done in the current procedure/labs scripts).